views/components/info/control.es

```diff
@@ -24,10 +25,10 @@ config.on('config.set', (path, value) => {
 })
 
 const PoiControl = connect((state, props) => ({
-  muted: get(state, 'config.poi.content.muted', false),
+  muted: get(state, 'config.poi.content.muted', false)
```

Comment:
Use a text editor with eslint (atom, vscode, sublime, etc) or run `./node_modules/.bin/eslint <filename>` in project root path to review code using current eslint rules.